argmin/src/solver/neldermead/mod.rs

```diff
@@ -12,6 +12,9 @@
 //!
 //! See [`NelderMead`] for details.
 //!
+//! A simple example of usage can be found at
+//! <https://github.com/argmin-rs/argmin/blob/main/argmin/examples/neldermead-simple.rs>
+//!
```

Comment:
In theory, I am all in favor of advertising the example here. The problem however is, that this points to the current main branch. If people use older versions of argmin (and the corresponding docs) they may be led to an example which doesn't work for them. We have had this in the past, and decided to remove all links to examples in the docs. One could link to the corresponding example via the git tags; however, it is a major chore to update all links for every new version.